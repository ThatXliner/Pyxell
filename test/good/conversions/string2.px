
n = -8251035946488047
print 0.toString() + n.toString()
