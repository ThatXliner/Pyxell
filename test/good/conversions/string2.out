0-8251035946488047
-9223372036854775808 9223372036854775807
